data/human/news.txt

```diff
@@ -752,6 +820,7 @@ news "hyperspace relay engineer"
 			`"Do you know the DRQ of the HSBE for the hyperlanes around here? You don't understand? Sorry, a hyperlane can be thought of as a corridor through space..." The engineer gives a long-winded and confusing explanation of hyperlanes.`
 			`"It might take hours to get a message a few systems away going through the hyperspace relays, but it sure beats taking multiple days to send a ship or multiple centuries to send a message at light speed."`
 			`"Someone with more money than sense asked me to refill the hyperspace relays, because their signal was low."`
+			`"Trust me, I'm an engineer."`
```

Comment:
Trust me to do what?

Reply:
Right, this doesn't work stand-alone. I took it out.